src/entity.unit.ts

```diff
@@ -533,6 +534,16 @@ describe("Entity", () => {
 			instance.update({ Skills: [skillInstance] }, null, true);
 			expect(instance.Skills.length).toBe(1);
 		});
+
+		it("only runs default rule for list properties onInitNew", async () => {
+			const model = new Model(PersonWithSkillsModel);
+			let instance = await model.types.Person.create({}) as any;
+			expect(instance.Skills.length).toBe(2);
+			instance = await model.types.Person.create({ Skills: [] }) as any;
+			expect(instance.Skills.length).toBe(2);
+			instance = await model.types.Person.create({ Id: "test", Skills: [] }) as any;
```

Comment:
This needs to not have `Skills` in the initial state.

Reply:
Removed